tests/test_models.py

```diff
@@ -75,6 +75,18 @@ def test_create_customer(self) -> None:
         self.assertEqual(customer.id, customer_fetched.id)
         return
 
+    def test_read_customer(self):
+        """Test reading a customer record"""
+        customer = CustomerFactory()
+        logging.debug(customer)
+        customer.id = None
+        customer.create()
+        self.assertIsNotNone(customer.id)
+        # Fetch it back
+        found_customer = Customer.find(customer.id)
```

Comment:
can you change this to Customer.find_or_404 so that its testing the new code that you added to the model?

Reply:
Done